Samurai/Components/ApiManager.cs

```diff
@@ -1,13 +1,10 @@
-using System;
 using System.Collections.Generic;
-using System.Net.Http;
 using System.Threading.Tasks;
+using AcidChicken.Samurai.Models;
```

Comment:
All of the local namespace reference instructions have to be included in the local namespace block.